harness/determined/cli/user.py

```diff
@@ -69,10 +70,14 @@ def log_in_user(parsed_args: Namespace) -> None:
     else:
         username = parsed_args.username
 
-    message = "Password for user '{}': ".format(username)
-
+    if "DET_PASS" in os.environ:
+        print("Using the enviornment variable DET_PASS for a user's password")
+        password = os.environ["DET_PASS"]
+    else:
+        message = "Password for user '{}': ".format(username)
```

Comment:
another nit: when you touch old `.format()` coded, can you update it to use f-strings?